packages/generators/generators/lib/files/ts/plugin/admin/src/components/PluginIcon/index.tsx

```diff
@@ -5,7 +5,7 @@
  */
 
 import React from 'react';
-import Puzzle from '@strapi/icons/Puzzle';
+import Puzzle from '@strapi/icons';
```

Comment:
Shouldn't this be `{ Puzzle }`

Reply:
ugh, dumb mistake - bad copy/paste - fixed now